src/java.base/share/classes/java/lang/Shutdown.java

```diff
@@ -174,6 +174,10 @@ static void exit(int status) {
      */
     private static void logRuntimeExit(int status) {
         try {
+            if (!VM.isBooted()) {
```

Comment:
Can you move it to the top of the method so that the isBooted check isn't in the try-catch block? isBooted shouldn't throw but if it did then we wouldn't want the logging exception handler running.

Reply:
Done - I've updated the PR with this change.